pkg/process/exec_conf.go

```diff
@@ -326,7 +326,11 @@ func cleanup(cmd *cobra.Command) {
 			if err != nil {
 				return err
 			}
-			err = present.SpansToSVG(fh, collect.CollectSpans(ctx, work))
+			if strings.HasSuffix(*traceOut, ".json") {
```

Comment:
maybe switch on the `filepath.Ext`?

Reply:
i believe we have some folks already who are writing svgs out without an svg extension, so i didn't want to break those use cases